### PR TITLE
chore: add parameter descriptions

### PR DESCRIPTION
**Pull request by tenthirtyam**
In order to have a good experience with our community, we recommend that you read the [contributing guidelines](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/blob/main/CONTRIBUTING_DCO.md) for making a pull request.

**Summary of Pull Request**

- Added parameter descriptions to be used for upcoming documentation.
- Trimmed trailing whitespace, where applicable.
- Adjusted spacing, where applicable.
- Updated module version from v1.0.0.1001 to v1.0.0.1002.

**Type of Pull Request**

- [ ] This is a bug fix.
- [ ] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [ ] This is a documentation update.
- [ ] This is a refactoring update.
- [x] This is a chore update
- [ ] This is something else.
      Please describe:

**Related to Existing Issues**

Issue Number: N/A

**Test and Documentation Coverage**

- [ ] Tests have been completed (for bug fixes / features).
- [ ] Documentation has been added / updated (for bug fixes / features).

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: VMware.CloudFoundation.CertificateManagement.psd1 [hunk 1]
```diff
@@ -10,123 +10,122 @@
...
     # Company or vendor of this module
     CompanyName = 'VMware Inc.'
-    
+
     # Copyright statement for this module
     Copyright = '(c) 2023 VMware All rights reserved'
-    
+
     # Description of the functionality provided by this module
     Description = 'A PowerShell module for VMware Cloud Foundation certificate management.'
```
Comment (GaryJBlake) [inline VMware.CloudFoundation.CertificateManagement.psd1:33]: Remove the full stop from this description, looks better with out

**Review by GaryJBlake — CHANGES_REQUESTED** [review]

File: VMware.CloudFoundation.CertificateManagement.psd1 (hunk 1 above)
Comment (tenthirtyam) [reply VMware.CloudFoundation.CertificateManagement.psd1:33]: Updating to 'PowerShell Module for VMware Cloud Foundation Certificate Management'.

Comment (tenthirtyam) [reply VMware.CloudFoundation.CertificateManagement.psd1]: Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/15/commits/7569fa212ab76b01d00e451ea281d23a8e49a337.

**Review by GaryJBlake — APPROVED** [review]
LGTM!

**Pull request merged**